src/textual/widgets/_text_area.py

```diff
@@ -684,26 +678,22 @@ def _watch_show_vertical_scrollbar(self) -> None:
         self._rewrap_and_refresh_virtual_size()
         self.scroll_cursor_visible()
 
-    def _watch_theme(self, theme: str | None) -> None:
+    def _watch_theme(self, theme: str) -> None:
         """We set the styles on this widget when the theme changes, to ensure that
-        if padding is applied, the colours match."""
+        if padding is applied, the colors match."""
         self._set_theme(theme)
 
-    def _app_dark_toggled(self) -> None:
+    def _app_dark_toggled(self):
```

Comment:
Why did you _remove_ the return type hint?

Reply:
Merge conflict mistake

Reply:
That's what I imagined :)